README.md

```diff
@@ -145,6 +145,7 @@ Some of them are described below:
   - Password field names
   - URL field names
   - OTP Auth field names
+  - Filter fields out by regex (blacklist)
```

Comment:
Huh, I'm quite tired, but I'm pretty sure we are dealing with objects called filenames, password names, passwords or items.
Fields are found in the password data ("login" "url "http" are commonly used fields)

Reply:
You weren't tired, but I was. Thanks for all the feedback. I'll probably go over my changes once more tomorrow.